mlir/include/mlir/Dialect/Arith/IR/ArithOps.td

```diff
@@ -457,8 +467,7 @@ def Arith_CeilDivUIOp : Arith_IntBinaryOp<"ceildivui",
     leading bit as the most significant, i.e. for `i16` given two's complement
     representation, `6 / -2 = 6 / (2^16 - 2) = 1`.
 
-    Note: the semantics of division by zero is TBD; do NOT assume any specific
-    behavior.
+    Division by zero is undefined behaviour.
```

Comment:
'The behavior is undefined when dividing by zero. Note that for `vector` and `tensor` operands, dividing any element by zero leads to undefined behavior.'